src/spicedb-common/services/developerservice.ts

```diff
@@ -226,16 +226,13 @@ class DeveloperServiceRequest {
       ),
     });
 
-    const developerRequest = JSON.stringify(
-      toJson(DeveloperRequestSchema, request),
-    );
+    const developerRequest = toJsonString(DeveloperRequestSchema, request);
     const encodedResponse: string =
       window[ENTRYPOINT_FUNCTION](developerRequest);
 
-    const response = create(
-      DeveloperResponseSchema,
-      JSON.parse(encodedResponse),
-    );
```

Comment:
This was the problem - the value that comes back from the WASM service has a string value for one of its enum values, and typescript (and by extension the underlying JS) were treating those enums as ints. Using `fromJsonString` coerces that value to the correct enum value, which means that you can then compare to a static enum value. [This check in particular](https://github.com/authzed/playground/blob/c1c5768b38a10a505aa7a98fe48ea8d0c2877590/src/services/check.ts#L135) was the one that was failing, because with the old code the LHS of that comparison was the string `"MEMBER"` instead of the numeric enum value.